README.md

```diff
@@ -5,7 +5,38 @@
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 [![Join the chat at https://gitter.im/poetapp/Lobby](https://badges.gitter.im/poetapp/Lobby.svg)](https://gitter.im/poetapp/Lobby)
 
-Po.et JS is a small library that provides methods to easily create and sign Po.et Claims. 
+Po.et JS is a small library that provides methods to easily create and sign Po.et Claims according to the 
+[Verifiable Credentials Data Model](https://w3c.github.io/vc-data-model). These claims are [JSON-LD](https://w3c.github.io/json-ld-syntax/)
+documents. As such, you can define your own JSON-LD `@context` to map your submitted Claims.
+
+Po.et does provide a default `@context object` that you can extend or override in the `createClaim` function. The current defaults are as follows:
+
+```ts  
+'@context': {
+   cred: 'https://w3id.org/credentials#',
+   schema: 'http://schema.org/',
+   sec: 'https://w3id.org/security#',
+   
+   // Verifiable Credentials (Claim) metadata
+   issuer: 'cred:issuer',
+   issuanceDate: 'cred:issued',
+   type: 'http://schema.org/additionalType',
+   claim: 'http://schema.org/Thing', // The most generic definition in schema.org
+   
+   // Work Claim Defaults
+   author: 'schema:author',
+   dateCreated: 'schema:dateCreated',
+   datePublished: 'schema:datePublished',
+   name: 'schema:name',
+   keywords: 'schema:keywords',
+   text: 'schema:text',
```

Comment:
Shouldn't we add the `archiveURL` and `canonicalUrl` to these defaults? I realize these are custom extensions, but isn't that OK, since this is also a custom context?